purchase_deposit/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Elico Corp, Dominique K. <dominique.k@elico-corp.com.sg>
+# Copyright 2019 Ecosoft Co., Ltd., Kitti U. <kittiu@ecosoft.co.th>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Purchase Deposit',
+    'version': '12.0.0.0.0',
```

Comment:
@kittiu `12.0.1.0.0`

Reply:
This is new module not migration, do I we need .1?

Reply:
Yes, it need :p. Will fix it.